### PR TITLE
add syndica case study

### PR DESCRIPTION
**Pull request by argshook**
This PR adds a new page [`/case-study/syndica`](https://deploy-preview-1534--unruffled-blackwell-31bfb2.netlify.app/case-study/syndica)

### CODE REVIEW

**Comment by tris0laris** [comment]
We're missing the new section under "customers"

File: src/pages/case-study/syndica.tsx [hunk 1]
```diff
@@ -0,0 +1,497 @@
...
+      alt: "Flag icon",
+      src: "/img/pages/case-study/icons/flag.svg",
+    },
+  },
+]
+
+const Page = () => {
+  const title = "Syndica"
+  const description =
+    "QuestDB is at the core of the observability and analytics capabilities for both Syndica's service to power the Web3 infrastructure and its internal monitoring."
```
Comment (tris0laris) [inline src/pages/case-study/syndica.tsx:98]: "QuestDB is at the core of the observability and analytics capabilities for both Syndica's service to power the Web3 infrastructure and its internal monitoring."

Change to: 

QuestDB is the database for real-time analytics and time-series dashboards at Syndica

File: src/pages/case-study/syndica.tsx [hunk 2]
```diff
@@ -0,0 +1,497 @@
...
+              juCss.jumbotron__description,
+            )}
+          >
+            {description}
+          </p>
+        </div>
+
+        <div className={juCss.jumbotron__banner}>
+          <Image
+            description="QuestDB is at the core of the observability and analytics capabilities for both Syndica's service to power the Web3 infrastructure and its internal monitoring."
```
Comment (tris0laris) [inline src/pages/case-study/syndica.tsx:140]: change the following: QuestDB is at the core of the observability and analytics capabilities for both Syndica's service to power the Web3 infrastructure and its internal monitoring.

to

Syndica powers the Web3 infrastructure

File: src/pages/case-study/syndica.tsx (hunk 1 above)
Comment (argshook) [reply src/pages/case-study/syndica.tsx:98]:
```suggestion
    "QuestDB is the database for real-time analytics and time-series dashboards at Syndica"
```

File: src/pages/case-study/syndica.tsx (hunk 2 above)
Comment (argshook) [reply src/pages/case-study/syndica.tsx:140]:
```suggestion
            description="Syndica powers the Web3 infrastructure"
```

**Pull request merged**